modules/client/src/controllers/WithdrawalController.ts

```diff
@@ -47,12 +47,14 @@ export class WithdrawalController extends AbstractController {
 
         await this.connext.watchForUserWithdrawal();
 
-        this.log.info(`Node Withdraw Response: ${JSON.stringify(transaction, replaceBN, 2)}`);
+        this.log.info(`Node Withdraw Response: ${stringify(transaction)}`);
       } else {
         this.log.info(`Calling ${CFCoreTypes.RpcMethodName.WITHDRAW}`);
-        const withdrawResponse = await this.connext.cfWithdraw(amount, assetId, recipient);
-        this.log.info(`Withdraw Response: ${JSON.stringify(withdrawResponse, replaceBN, 2)}`);
-        transaction = await this.ethProvider.getTransaction(withdrawResponse.txHash);
+        // Bo note: the following function used to point to something that doesn't exist
+        // so this code path probably never gets executed.. idk what this was supposed to do
+        const withdrawResponse = await this.node.withdraw(transaction as any);
```

Comment:
this was to allow the user to submit their own withdrawal, without it going through the hub to pay gas. we should probably keep it in here :)